src/Simulation/EntryPointDriver/Azure.cs

```diff
@@ -48,9 +50,35 @@ internal static class Azure
             }
             else
             {
-                var job = await machine.SubmitAsync(
+                try
+                {
+                    var job = await machine.SubmitAsync(
                     entryPoint.Info, input, new SubmissionContext { Shots = settings.Shots });
-                DisplayJob(job, settings.Output);
+                    DisplayJob(job, settings.Output);
+                }
+                catch (AzureQuantumException azureQuantumEx)
+                {
+                    DisplayWithColor(
+                        ConsoleColor.Red,
+                        Console.Error,
+                        "Something went wrong related to Azure quantum.");
```

Comment:
IMO, this message could describe a translation error as well, since for the user, the entire process is related to Azure Quantum. This is a little more specific:

```suggestion
                        "Something went wrong when submitting the program to the Azure Quantum service.");
```

Reply:
Yes, you are right, I will change it such that both have the same message.

Reply:
Thinking a little bit more about it, I think it is ok to have to separate messages since one indicates a problem with service related things which can be credentials, unavailability and the other one is a problem with the program itself.

Reply:
Yes, having different messages for translation and service errors is fine - just wanted to make the service error message more specific.